client/src/app.jsx

```diff
@@ -15,10 +15,25 @@ class App extends React.Component {
       prices: Sample.prices,
       high: 0,
       low: 0,
+      priceRange: 0,
       ratingPercent: `81%`,
       peopleOwn: 2500,
-      path: 'M0 300 L10 290'
+      path: 'M0 300'
     }
+
+    this.updateTimeFrame = this.updateTimeFrame.bind(this);
+    this.calculatey = this.calculateY.bind(this);
```

Comment:
is this bound? should that be a capital y in 'this.calculatey'?

Reply:
Yes, that was supposed to be calculateY. Removed the binding (unnecessary because the function isn't getting passed to another component)